apps/rush-lib/src/cli/actions/VersionAction.ts

```diff
@@ -93,13 +93,18 @@ export class VersionAction extends BaseRushAction {
   }
 
   protected run(): Promise<void> {
-    return Promise.resolve().then(() => {
+    return Promise.resolve().then(async () => {
```

Comment:
Should be able to make this whole function async.

Reply:
This would come in my tab-completion PR.